File: Leetcode/Easy/7-Fizz-Buzz/solution.py
class Solution(object):
    def fizzBuzz(self, n):
        arr = []
        for i in range(1,n+1):
            div_5 = (i % 5 == 0)
            div_3 = (i % 3 == 0)

            if div_5 and div_3:
                arr.append("FizzBuzz")
            elif div_3:
                arr.append("Fizz")
            elif div_5:
                arr.append("Buzz")
            else:
                arr.append(str(i))
        return arr
                

File: Leetcode/Easy/7-Fizz-Buzz/solution.go
package main

import "fmt"

func fizzBuzz(n int) []string {
	arr := []string{}
	for i := 1; i < n+1; i++ {
		if i%5 == 0 && i%3 == 0 {
			arr = append(arr, "FizzBuzz")
		} else if i%3 == 0 {
			arr = append(arr, "Fizz")
		} else if i%5 == 0 {
			arr = append(arr, "Buzz")
		} else {
			arr = append(arr, fmt.Sprintf("%d", i))
		}
	}
	return arr
}
